site/docs/components/AppDemo.tsx

```diff
@@ -0,0 +1,120 @@
+'use client';
+import { OnchainKitProvider } from '@coinbase/onchainkit';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import type { ReactNode } from 'react';
+import { http, WagmiProvider, createConfig } from 'wagmi';
+import { base, baseSepolia } from 'wagmi/chains';
+import { coinbaseWallet } from 'wagmi/connectors';
+import '@coinbase/onchainkit/styles.css';
+import { createWalletClient } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { useTheme } from '../contexts/Theme.tsx';
+
+const queryClient = new QueryClient();
+
+// Default value to avoid build errors when running locally.
+const DEFAULT_DEMO_PK =
+  '0x1234567890123456789012345678901234567890123456789012345678901234';
+
+const demoWalletConnector = () => () => {
+  // Demo account with no real assets
+  const demoWalletPK = import.meta.env.VITE_DEMO_WALLET_PK || DEFAULT_DEMO_PK;
```

Comment:
Demo accounts with no value attached. We could just hard code VITE_DEMO_WALLET_PK